reverse.js

```diff
@@ -24,7 +24,8 @@
     d = d.replace(/,/g,' ')
          .replace(/([^eE])-/g,'$1 -')
          .replace(/\s*([achlmqstvzACHLMQSTVZ])\s*/g," $1 ")
-         .replace(/\s+/g, ' ');
+         .replace(/\s+/g, ' ')
+         .replace(/(\d*\.\d+([eE]-?\d+)?)((\.\d*)+)/g, (_, g1, g2, g3) => g1 + g3.replaceAll(".", " ."));
```

Comment:
One last thing here: `((\.\d*)+` should be `((\.\d+)+` so we don't try to turn patterns like `4...` into `4 . . .`

Reply:
Good point, I updated the regular expression